src/main.rs

```diff
@@ -38,6 +38,7 @@ extern crate log;
 /// If you're looking for the actual scanning, check out the module Scanner
 fn main() {
     env_logger::init();
+    let startprocess = Instant::now();
```

Comment:
The Rust standard is snake_case, please rename to start_process :)

Reply:
Corrected